samples/java/tutorial_code/video/optical_flow/OpticalFlowDenseDemo.java

```diff
@@ -0,0 +1,72 @@
+import java.util.ArrayList;
+import org.opencv.core.*;;
+import org.opencv.highgui.HighGui;
+import org.opencv.imgproc.Imgproc;
+import org.opencv.video.Video;
+import org.opencv.videoio.VideoCapture;
+
+
+class OptFlowDense {
+    public void run() {
+        String filename = "../data/vtest.avi";
```

Comment:
Consider using:
- command line's "args"
- `findFileOrKeep()` helper function for samples (from `Core`)

Reply:
I kept the name like that because the following [code](https://github.com/opencv/opencv/tree/master/samples/java/tutorial_code/video/background_subtraction) for background subtraction has its name written like that